mizar/dp/mizar/operators/endpoints/endpoints_operator.py

```diff
@@ -164,6 +165,23 @@ def update_scaled_endpoint_backend(self, name, spec):
                 for a in s['addresses']:
                     backends.add(a['ip'])
         ep.set_backends(list(backends))
+        ports = {}
+        service = kube_get_service(self.core_api, name, namespace)
+        if not service or not service.metadata or not service.metadata.annotations:
+            return
+        service_spec = list(service.metadata.annotations.values())
+        json_spec = json.loads(service_spec[0])
+        # port = {frontend_port: [backend_port, protocol]}
```

Comment:
This is just a comment showing how the port dictionary maps the frontend port to a backend port and protocol.

Reply:
This was a bit confusing to me is because in the following lines,  you have: ports[port["port"]] = [] <-- this port dict is from spec
I think this dictionary maps you listed here is port(s) instead of port, right?

Reply:
Ah I see what you mean. Yeah, I'll change the name to make the correspondence more clear.